app/templates/content/suppliers-guide-g-cloud.html

```diff
@@ -205,26 +220,13 @@ <h3 id="live">
   </p>
 
   <h2 id="when-to-apply">
-  <span class="number">3. </span>When to bid</h2>
+  <span class="number">4. </span>When to bid</h2>
 
   <p>A new version of the G-Cloud framework is released about every 6 months. Each version of the framework runs for 12 months so frameworks will overlap. </p>
 
-  <h3 id="new-suppliers">
-  <span class="number">3.1 </span>New suppliers</h3>
-
-  <p>
-  All cloud services on the Digital Marketplace are part of the G-Cloud framework – a catalogue of services where the basic terms of use have already been agreed between the government and suppliers. This means buyers can buy services without running a full tender process.
-  </p>
-
-  <p>
-  If you’re new to G-Cloud you can bid to supply services when a new version is published on the Official Journal of the European Union (<abbr title="Office Journal of the European Union">OJEU</abbr>). You don’t need to be based in the UK to apply to the framework, but you need to agree to the terms of the framework agreement and call-off contract, which are governed by English law.
-  </p>
-
   <p>Follow the steps in ‘<a href="#how-to-apply">How to bid</a>’ to submit a new application. </p>
 
-
-  <h3 id="existing-suppliers">
-  <span class="number">3.2 </span>Existing suppliers</h3>
+  <h3 Existing suppliers</h3>
```

Comment:
It would be good to keep the `existing-suppliers` id so it's still possible to link to this section of the document.